logger/src/main/java/com/orhanobut/logger/Logger.java

```diff
@@ -303,8 +303,21 @@ private static void logHeaderContent(int logType, String tag, int methodCount) {
             logDivider(logType, tag);
         }
         String level = "";
+
+        int stackOffset = -1;
+        boolean foundThisClass = false;
```

Comment:
would be better to create another method in order to determine stack offset. and I would also suggest another approach to find the correct stack index.

We can iterate by starting from 3 (there is always 2 stack and sometimes another stack comes from somewhere that I couldn't find the real reason). We guarantees that starting trace will be a logger method. Then we simplify the method.
```java
    private static int getStackIndex(StackTraceElement[] trace) {
        for (int i = 3; i < trace.length; i++) {
            StackTraceElement e = trace[i];
            if (!e.getClassName().equals(Logger.class.getName())) {
                return --i;
            }
        }
        return -1;
    }
```